### PR TITLE
CycleGAN and Download/Resize scripts

### PR DESCRIPTION
**Pull request by ezraschwaa**
Sorry about not making each change one pull request--will do so in the future.

Just adds the three files to the repo and updated the readme

@mhrmm

### CODE REVIEW

File: resizescript.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from PIL import Image
+import os,sys
+
+path = "/Users/ezraschwartz/Documents/thesis/images/"
```
Comment (mhrmm) [inline resizescript.py]: Don't hardcode these paths. Make them arguments to the script.

File: downloadscript.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#run as
```
Comment (mhrmm) [inline downloadscript.py]: Move both scripts into a scripts folder.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,18 @@
 # thesis-ezra-schwartz
 Ezra Schwartz's thesis project
 
-Hello world
\ No newline at end of file
+downloadscript: 
+
+run python downloadscript.py <url csv filename>
```
Comment (mhrmm) [inline README.md]: Put four spaces in front, so that Markdown formats this as code.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,18 @@
...
+
+program will download all images from a given url and put them into a folder called images
+
+resizescript:
+
+run python resizescript.py
```
Comment (mhrmm) [inline README.md]: Put four spaces in front, so that Markdown formats this as code.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,18 @@
...
+
+program will take all images from your ./images folder and put 128 x 128 versions in your ./resized_images folder
+
+Drive folder with directories of data:
```
Comment (mhrmm) [inline README.md]: Make this a small heading.

File: README.md [hunk 6]
```diff
@@ -1,4 +1,18 @@
 # thesis-ezra-schwartz
 Ezra Schwartz's thesis project
 
-Hello world
\ No newline at end of file
+downloadscript:
```
Comment (mhrmm) [inline README.md]: Make this a small heading, i.e. 

### downloadscript

File: README.md [hunk 7]
```diff
@@ -1,4 +1,18 @@
...
 
-Hello world
\ No newline at end of file
+downloadscript: 
+
+run python downloadscript.py <url csv filename>
+
+program will download all images from a given url and put them into a folder called images
+
+resizescript:
```
Comment (mhrmm) [inline README.md]: Make this a small heading.

**Review by mhrmm — COMMENTED** [review]
Good, just some minor suggested changes.

File: resizescript.py (hunk 1 above)
Comment (ezraschwaa) [reply resizescript.py]: Fixed this

**Review by ezraschwaa — COMMENTED** [review]
@mhrmm

**Review by mhrmm — COMMENTED** [review]
Remove the two scripts from the top-level dir.

**Pull request merged**